racing_opt.jl

```diff
@@ -0,0 +1,86 @@
+using JuMP
```

Comment:
@dfridovi, I am starting just an outline of what the optimization through Julia would look like. I tried to follow the docs here: https://jump.dev/JuMP.jl/stable/tutorials/Nonlinear%20programs/rocket_control/

Happy to take any suggestions, even for the rest of the code. Although I realize it has 0 documentation and probably would make no sense, if something really stands out it probably means there's an issue.